src/App.tsx

```diff
@@ -1,61 +1,193 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
 import './App.scss';
+import 'bulma/css/bulma.css';
+
+import { TodoList } from './components/TodoList';
+
```

Comment:
Consider moving `User` and `Todo` types to separate files for better organization. 📁